src-v5/slide.tsx

```diff
@@ -168,7 +168,7 @@ const Slide = ({
         );
       }
     }
-  }, [isVisible]);
+  }, [adaptiveHeight, frameHeight, isVisible, setFrameHeight]);
```

Comment:
I would need to test this more, otherwise great one. Thanks for it

Reply:
Thanks, I updated those deps too. Didn't spot those warnings initially..